legacy/src/app/partials/nodes-list.html

```diff
@@ -715,10 +715,15 @@ <h4 class="page-header__dropdown-title">Add device</h4>
             <h4 class="page-header__dropdown-title">Add rack controller</h4>
             <pre><code># To add a new rack controller, SSH into the rack controller.
 # Install the maas-rack-controller package.
+# Please confirm maas version is the same as main rack controller first.
```

Comment:
Please change the text in line 718 to:
'Confirm that the maas version is the the same is the main rack controller.'